system/lib/wasmfs/syscalls.cpp

```diff
@@ -1019,4 +1017,8 @@ long __syscall_access(long path, long amode) {
 
   return 0;
 }
+
+long __syscall_access(long path, long amode) {
```

Comment:
It seems odd that we would want/need to support both of the syscalls...

Reply:
I looks like we can/should make PR to remove our definition of __syscall_access so musl will just use __syscall_accessat

Reply:
Ok, I'll try that.